### DISCUSSION TITLE
Save to file - Load from file?

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#613] [Q&A]
What do the "Save to file" and "Load from file" buttons do? There's no mention of them in the bwc-manual.pdf and when I press them I get no indication that they are doing anything. 

-Wes

[Image: "Save to file" and "Load from file" buttons](http://www.westhaver.com/save_load.png)

**Comment by visualapproach** [comment]
They save and load the command queue to/from file. In your case the buttons are disabled because the functions was not written yet on that version.
